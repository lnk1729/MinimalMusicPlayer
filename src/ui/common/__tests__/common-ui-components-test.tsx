import React from 'react';
import {View} from 'react-native';
import renderer from 'react-test-renderer';
import Header from '../header.component';
import Screen from '../screen.component';

test('Header renders correctly', () => {
  const tree = renderer.create(<Header title="Mock title" />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Screen renders correctly', () => {
  const tree = renderer
    .create(
      <Screen hasPadding={false}>
        <View />
      </Screen>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
