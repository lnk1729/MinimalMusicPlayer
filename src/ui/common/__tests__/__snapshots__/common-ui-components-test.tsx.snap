// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "paddingVertical": 12,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#1DB954",
        "fontSize": 30,
        "fontWeight": "600",
      }
    }
  >
    Mock title
  </Text>
</View>
`;

exports[`Screen renders correctly 1`] = `
<RNCSafeAreaView
  style={
    Array [
      Object {
        "backgroundColor": "#333333",
        "flex": 1,
        "paddingHorizontal": 20,
      },
      Object {
        "paddingHorizontal": 0,
      },
    ]
  }
>
  <View />
</RNCSafeAreaView>
`;
